### PR TITLE
[FIX] website_slides: fix bs4 bugs

### PR DESCRIPTION
**Pull request by seb-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by rdeodoo** [comment]
hi @qsm-odoo can you check this? (Check the diff with `?w=1` it will be easier to read)
UI looks fine, there is a last fix on iframe/canvas scrollbar to check with seb

File: addons/website_slides/static/src/js/slides_embed.js [hunk 1]
```diff
@@ -103,6 +103,8 @@ $(document).ready(function () {
             fullScreenFooter: function (ev) {
                 if (ev.target.id === "PDFViewerCanvas") {
                     this.pdf_viewer.toggleFullScreenFooter();
+                    // debugger;
```
Comment (qsm-odoo) [inline addons/website_slides/static/src/js/slides_embed.js]: to remove :)

File: addons/website_slides/static/src/scss/website_slides.scss [hunk 2]
```diff
@@ -16,7 +16,7 @@ $MAX-Z-INDEX : 2147483647 !default;
 }
 
 .oe_slides_panel_footer {
-    background-color: $light;
+    width: 100%;
```
Comment (qsm-odoo) [inline addons/website_slides/static/src/scss/website_slides.scss]: Why removing `container-fluid` on this element to add back `width: 100%;` here ?

File: addons/website_slides/views/website_slides.xml [hunk 3]
```diff
@@ -662,12 +660,12 @@
 
 <!-- Slide sub-template: display an item in a list of related slides (Related, Most Viewed, ...) -->
 <template id="related_slides" name="Related Slide">
-    <li class="media">
-        <a t-attf-href="/slides/slide/#{slug(slide)}">
+    <li class="media mt16">
+        <a t-attf-href="/slides/slide/#{slug(slide)}" class="mr16">
```
Comment (qsm-odoo) [inline addons/website_slides/views/website_slides.xml]: Use BS `m*-*` classes (same for all other ones you added)

File: addons/website_slides/views/website_slides_embed.xml [hunk 4]
```diff
@@ -31,119 +32,109 @@
...
-                    <div t-if="is_embedded" class="navbar-default oe_slides_share_bar">
-                        <div class="container-fluid">
-                            <div class="row">
-                                <div class="col-xl-8 col-lg-8 col-md-4 col-4 text-left">
-                                    <div class="oe_slides_ellipsis">
-                                        <a target="_new" t-att-href="slide.website_url">
-                                            <span  t-esc="slide.name" t-att-title="slide.name"/>
-                                        </a>
+                <body t-if="not private">
+                    <div id="PDFViewer" class="border" style="height: 100%;">
```
Comment (qsm-odoo) [inline addons/website_slides/views/website_slides_embed.xml]: `h-100`

File: addons/website_slides/views/website_slides_embed.xml [hunk 5]
```diff
@@ -31,119 +32,109 @@
...
+                                    <div class="col-lg-1 col-2 text-right">
+                                        <b class="oe_slide_js_embed_option_link" data-slide-option-id="#slide_embed"><i class="fa fa-code"></i> Embed</b>
                                     </div>
                                 </div>
-                                <div class="col-xl-1 col-lg-1 col-md-2 col-2 text-right">
-                                    <b class="oe_slide_js_embed_option_link" data-slide-option-id="#slide_share"><i class="fa fa-share-alt"></i> Share</b>
+                            </div>
+                        </div>
+                        <!--  PDF Viewer Body : contains the canvas, the loader, or the image-->
+                        <div id="PDFSlideViewer" class="text-center" style="position:relative; flex-grow: 1; flex-shrink: 1;"
```
Comment (qsm-odoo) [inline addons/website_slides/views/website_slides_embed.xml]: Use BS utilities to do that if it is really needed

**Comment by seb-odoo** [comment]
@qsm-odoo thank you for the review, I made the changes (and some other fixes). If you could quickly check again while I wait for green runbot... Thanks!

File: addons/website_slides/views/website_slides_embed.xml [hunk 6]
```diff
@@ -31,119 +32,109 @@
...
-                    <!-- PDF Viewer Header : contains the name, and the share links -->
-                    <div t-if="is_embedded" class="navbar-default oe_slides_share_bar">
-                        <div class="container-fluid">
-                            <div class="row">
-                                <div class="col-xl-8 col-lg-8 col-md-4 col-4 text-left">
-                                    <div class="oe_slides_ellipsis">
-                                        <a target="_new" t-att-href="slide.website_url">
-                                            <span  t-esc="slide.name" t-att-title="slide.name"/>
-                                        </a>
+                <body t-if="not private">
```
Comment (qsm-odoo) [inline addons/website_slides/views/website_slides_embed.xml]: t-else=""

**Pull request merged**